tractor/_debug.py

```diff
@@ -132,19 +130,6 @@ async def _acquire_debug_lock(uid: Tuple[str, str]) -> AsyncIterator[None]:
         log.error(f"TTY lock released by {task_name}:{uid}")
 
 
-def handler(signum, frame):
```

Comment:
Turns out we can just keep `trio` standard KBI handling as long as we prevent `pdb.Pdb` from overriding it!